src/lib/guesstimator/formatter/formatters/lib.js

```diff
@@ -54,18 +54,19 @@ export const normalTextMixin = Object.assign(
     },
     _matchesText(text) { return this._hasRelevantSymbol(text) },
     _normalTextErrors(text) {
-      let errs = []
-      if (this._inputSymbols(text).length > 1) { errs.push('Must contain only 1 symbol') }
-      if (!_.every(this._numbers(text), (e) => isParseableNumber(e))) { errs.push('Not all numbers are parseable') }
-      else if (this._numbers(text)[0] >= this._numbers(text)[1]) { errs.push('Low -> High') }
+      if (this._inputSymbols(text).length > 1) { return ['Must contain only 1 symbol'] }
 
-      return errs
+      const numbers = this._numbers(text)
+      if (numbers.length !== 2) { return ['Must contain 2 inputs'] }
+      if (!_.every(numbers, (e) => isParseableNumber(e))) { return ['Not all numbers are parseable'] }
+      if (numbers[0] >= numbers[1]) { return ['Low number must be first'] }
```

Comment:
We should format these like they are for users. So I would say, ['Format must be [low, high]']. Feel free to leave for later.